RevenueCatUI/PaywallView.swift

```diff
@@ -6,7 +6,6 @@ import SwiftUI
 @available(watchOS, unavailable, message: "RevenueCatUI does not support watchOS yet")
 @available(macOS, unavailable, message: "RevenueCatUI does not support macOS yet")
 @available(macCatalyst, unavailable, message: "RevenueCatUI does not support Catalyst yet")
-@MainActor
```

Comment:
I removed all these so users don't have to worry about making the places that use it `@MainActor` too.
I did it to handle `PurchaseHandler` being `@MainActor`, but I handled that internally instead so it doesn't "leak" through the API.